MANIFEST.in

```diff
@@ -1,3 +1,35 @@
 graft habitat/utils/visualizations/assets
+graft habitat/configs
 include habitat/py.typed
+
+include LICENSE
+
 include habitat_baselines/py.typed
+recursive-include habitat_baselines/config *.yaml
+
+exclude .circleci/config.yml
+exclude .editorconfig
+
+# added by check-manifest
+include *.md
+include *.txt
+include *.yaml
+include Dockerfile
+include mypy.ini
+recursive-include docs *.png
+recursive-include docs *.py
+recursive-include docs *.rst
+recursive-include docs *.sh
+recursive-include examples *.ipynb
```

Comment:
Some of these should be recursive-exclude probably.

Reply:
We shouldn't include docs and examples in the package IMO

Reply:
@erikwijmans Then feel free to recursive-exclude them. All check-manifest wants is all files to be covered by one of these globs (either an exclude/include or an ignore in its config.